Applications/LoRATrainingExample/ContentView.swift

```diff
@@ -131,30 +133,29 @@ class LoRAEvaluator {
     private let evaluateShowEvery = 8
     private let maxTokens = 200
 
-    private func loadModel() async throws -> (LLMModel, Tokenizer) {
+    private func loadModel() async throws -> ModelContainer {
```

Comment:
This is the same sort of transformation that LLMEval has -- use the new actor wrapper.